### PR TITLE
Feature/id 388/searching improvements and basic filters waypoint

### PR DESCRIPTION
**Pull request by JuanPabloArequipa**
**What?**

The searcher UI design was modified into a new text field which shows the input after searching any place and also it was made a message when there is not results found.

**Why?**

Now that the user can see the address searched into the text field

**How?**

The search UI was refactored and a message of Results not found, added.

Trainer
@mmoralesv

Code reviewers
@Elvis-Castro-H 
@aguirrekarina 
@saletG08

### CODE REVIEW

**Review by aguirrekarina — APPROVED** [review]

**Review by saletG08 — APPROVED** [review]
I don't have any comment

**Comment by Elvis-Castro-H** [comment]
Resolve merge conflicts

File: app/src/main/java/com/isc/hermes/model/WayPoint.java [hunk 1]
```diff
@@ -29,6 +29,11 @@ public WayPoint(String placeName, JsonObject properties, double latitude, double
         this.longitude = longitude;
     }
 
+    public WayPoint(String placeName, double latitude, double longitude) {
```
Comment (Elvis-Castro-H) [inline app/src/main/java/com/isc/hermes/model/WayPoint.java:39]: Missing javadoc

**Review by Elvis-Castro-H — APPROVED** [review]
Approved

**Pull request merged**